cookbook/core/flyte_basics/task.py

```diff
@@ -42,14 +45,63 @@ def square(n: int) -> int:
 
 # %%
 # In this task, one input is ``n`` which has type ``int``.
-# the task ``square`` takes the number ``n`` and returns a new integer (squared value)
+# The task ``square`` takes the number ``n`` and returns a new integer (squared value).
 #
 # .. note::
 #
-#   Flytekit will assign a default name to the output variable like ``out0``
+#   Flytekit will assign a default name to the output variable like ``out0``.
 #   In case of multiple outputs, each output will be numbered in the order
-#   starting with 0. For e.g. -> ``out0, out1, out2, ...``
+#   starting with 0, e.g., -> ``out0, out1, out2, ...``.
 #
-# Flyte tasks can be executed like normal functions
+# You can execute a Flyte task as any normal function. 
 if __name__ == "__main__":
     print(square(n=10))
+
+# %%
+# Single Task Execution
+# =====================
+# Tasks are the atomic units of execution in Flyte. 
+# Although workflows are traditionally composed of multiple tasks with dependencies defined by shared inputs and outputs, 
+# it can be helpful to execute a single task during the process of iterating on its definition. 
+# It can be tedious to write a new workflow definition every time you want to execute a single task under development 
+# but "single task executions" can be used to iterate on task logic easily.
+#
+# You can launch a task on Flyte console by inputting an IAM role or a Kubernetes service account.
+#
+# Alternatively, you can use ``flytectl`` to launch the task. Run the following commands in the ``core`` directory.
+#
+# Build a Docker image to package the task.
+#
+# .. prompt:: bash $
+#
+#   flytectl sandbox exec -- docker build . --tag "core:v1"
+#
+# Package the task.
+#
+# .. prompt:: bash $
+#
+#   pyflyte --pkgs flyte_basics package --image core:v1
+#
+# Register the task.
+#
+# .. prompt:: bash $
+#
+#   flytectl register files --project flytesnacks --domain development --archive flyte-package.tgz --version v1
+#
+# Generate an execution spec file.
+#
+# .. prompt:: bash $
+#
+#   flytectl get task --domain development --project flytesnacks flyte_basics.task.square --version v1 --execFile exec_spec.yaml
+#
+# Create an execution using the exec spec file.
+#
+# .. prompt:: bash $
+#
+#   flytectl create execution --project flytesnacks --domain development --execFile exec_spec.yaml
```

Comment:
probably also comment that - for subsequent executions the previous steps do not need to be repeated. 
Also mentions that you can use the console to launch the task?